tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, test_context):
         self.num_output_partitions = 3
         self.num_seed_messages = 100000
         self.transaction_size = 750
+        self.transaction_timeout = 10000
```

Comment:
During the test, I realized one scenario which is pretty dangerous. If we have a hard failure and the txn offsets get sticky on the broker side, for next consumer it could take over one minute to back-off and wait for the clearance, thus timing out on the fetch. Shrinking to 10 seconds here is just a remediation, but we should pay attention to general usage as well.